packages/core/src/components/widgets/SubmitButton.tsx

```diff
@@ -18,7 +18,7 @@ export default function SubmitButton<T, F>({
       <button
         type="submit"
         {...submitButtonProps}
-        className={`btn btn-info ${submitButtonProps.className}`}
+        className={submitButtonProps.className || `btn btn-info`}
```

Comment:
@jacqueswho Your change will replace the classNames entirely. Was that the expected behavior, rather than adding additional formatting to the className?

Reply:
Yes, so that is the default for core but if a user puts his own, there will be no clash or confusion. What do you think? They can still include then the default with their own

Reply:
Most of the other libraries that I've used (like Material UI) the classNames are added in addition to the default classes required to properly render the defaults. I'm worried that without the `btn btn-info` classes the resulting button would look very different. Most people will probably do things like move the button, change its color stuff like that.

Reply:
Ok will close pull request